tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -598,6 +650,32 @@ private static HttpResponseMessage CreateBearerAndPopHttpResponse(bool combinedC
             };
         }
 
+        private static HttpResponseMessage CreateMultipleHttpHeaderResponse()
+        {
+            return new HttpResponseMessage(HttpStatusCode.Unauthorized)
+            {
+                Headers =
+                {
+                    { WwwAuthenticateHeaderName, $"Bearer authorization_uri=\"https://login.microsoftonline.com/TenantId/oauth2/authorize\"" },
+                    { WwwAuthenticateHeaderName, $"Bearer authorization_uri=\"https://login.microsoftonline.com/TenantId/oauth2/authorize2\"" },
+                    { WwwAuthenticateHeaderName, $"PoP nonce=\"someNonce\"" },
+                    { WwwAuthenticateHeaderName, $"PoP nonce=\"someNonce2\"" }
+                }
+            };
+        }
+
+        private static HttpResponseMessage CreateUppercaseHttpHeaderResponse()
+        {
+            return new HttpResponseMessage(HttpStatusCode.Unauthorized)
+            {
+                Headers =
+                {
+                    { WwwAuthenticateHeaderName, $"Bearer AUTHORIZATION_URI=\"https://login.microsoftonline.com/TenantId/oauth2/authorize\"" },

```

Comment:
Can you also make the scheme name be uppercase, I think that's valid scenario?

Reply:
Cant do that for the scheme, it is case sensitive